views/partials/emptyModule.html

```diff
@@ -0,0 +1,8 @@
+<div class='module'>
+    <h1>{{ 'noContentLabel'|trans(current_locale.code) }}</h1>
```

Comment:
This should be a h2 / h3 not a h1 / h2 I believe.

Reply:
I stole this from the JS implementation.

Not sure what the proper value should be then.

Reply:
Looks like JS updated it a few days ago, and I had the older template.

Updating

Reply:
It should be h2, h3. I'll paste you the slack message deciding this.